### DISCUSSION TITLE
failed to init new actor (RetCode=16)

### DISCUSSION THREAD

**Discussion by colccool** [filecoin-project/lotus#6822] [Storage Provider]
when I run "lotus-miner --owner --worker ",the next bug appear,  Has anyone ever had this ### problem 

2021-07-21T20:52:59.214+0800 ERROR main lotus-storage-miner/init.go:261 Failed to initialize lotus-miner: creating miner failed:
  main.storageMinerInit
    /home/bitchina/lotus/cmd/lotus-storage-miner/init.go:541
 - pushing createMiner message:
  main.createStorageMiner
    /home/bitchina/lotus/cmd/lotus-storage-miner/init.go:713
 - GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: failed to init new actor (RetCode=16)
2021-07-21T20:52:59.214+0800 INFO main lotus-storage-miner/init.go:266 Cleaning up /home/bitchina/miner after attempt...

**Comment by chop8082** [comment]
Have you solved this problem?

**Comment by colccool** [comment]
no yet, i dont know why ，review the code,but no work






------------------ Original ------------------
From: chop8082 ***@***.***&gt;
Date: Sat,Jul 24,2021 10:41 PM
To: filecoin-project/lotus ***@***.***&gt;
Cc: sexfody ***@***.***&gt;, Manual ***@***.***&gt;
Subject: Re: [filecoin-project/lotus] failed to init new actor (RetCode=16) (#6822)





 
Have you solved this problem?

**Comment by LexLuthr** [comment]
I am archiving this discussion as it is a duplicate of https://github.com/filecoin-project/lotus/discussions/6863
